### ISSUE TITLE
Tab order update

### ISSUE THREAD

**Issue by johnwdubois** [johnwdubois/rezonator#1389]
1. Tab order (in main screen) should be updated to:
- Unit, Stack, Track, Resonance, Clique, Show, Tree, Chunk, Tag, Search
2. Change "Resonate" to "Resonance"
